src/components/cylc/TreeItem.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div>
+    <v-layout
+        row
+        v-show="depth >= minDepth"
+        :class="getNodeClass()"
+        :style="getNodeStyle()"
+    >
+      <!-- the node's left icon; used for expand/collapse -->
+      <v-flex
+        shrink
+        v-if="hasChildren"
+        class="type"
+        @click="typeClicked"
+        :style="getTypeStyle()"
+      >{{ isExpanded ? '&#9661;' : '&#9655;' }}</v-flex>
+      <!-- the node value -->
+      <v-layout @click="nodeClicked" row wrap v-if="node.__type === 'cyclepoint'">
```

Comment:
IMO yes. Though I am not sure how to do that in a simple way. So would move it for later. Other components (e.g. Task, Job) may benefit of this improvement too. We have one `Enum` for the Task States, but that's hardcoded too.

Perhaps another Enum, or a constant file somewhere. Will add a `TODO` so that I don't forget to look into it later and raise an issue and/or a fix for it.

Reply:
Done